src/reducer/wishlistReducer.js

```diff
@@ -0,0 +1,24 @@
+const wishlistReducer=(state,action)=>{
+
+    switch(action.type){
+        case "ADD_TO_WISHLIST":
+            return{
+                ...state,
+                wishlist:[...action.payload],
+                totalItems:action.payload.length
+            };
+        case "DELETE_FROM_WISHLIST":
+            return{
+                ...state,
+                wishlist:[...action.payload],
+                totalItems:action.payload.length
+            }
```

Comment:
You can try to combine these both into one function, where you check if the item already exits then you remove it, or else you add it to wishlist.

Reply:
for readability purpose i've seperated them